docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,277 @@
 title: Quickstart to run Loki locally
 menuTitle: Loki quickstart
 weight: 200
-description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+description: How to create and use a local Loki cluster for testing and evaluation purposes.
+killercoda:
+  preprocessing:
+    substitutions:
+      - regexp: evaluate-loki-([^-]+)-
+        replacement: evaluate-loki_${1}_
+  title: Loki Quickstart Demo
+  description: This sandbox provides an online enviroment for testing the Loki quickstart demo.
+  details:
+    finish:
+      text: finished.md
+  backend:
+    imageid: ubuntu
 ---
 
+<!-- Killercoda intro.md START -->
+
 # Quickstart to run Loki locally
 
 If you want to experiment with Loki, you can run Loki locally using the Docker Compose file that ships with Loki. It runs Loki in a [monolithic deployment](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode) mode and includes a sample application to generate logs.
 
-The Docker Compose configuration instantiates the following components, each in its own container:
+The Docker Compose configuration runs the following components, each in its own container:
+
+- **flog**: which generates log lines.
+
+  [flog](https://github.com/mingrammer/flog) is a log generator for common log formats.
```

Comment:
Having this on a separate line breaks the bulleted list in the output.

Reply:
Is that true? It looks fine for me:

<img width="1188" alt="Screenshot 2024-06-24 at 21 19 25" src="https://github.com/grafana/loki/assets/4599384/1d6aa382-71ca-4ac1-973a-23b26aa5a895">

Reply:
That's what I'm talking about.  I think both sentences should be on the same line in the output.

Reply:
Oh, right, the bulleted list is not broken but the blank line means that the list item has two paragraphs instead of one paragraph. I can make that change.

Reply:
Fixed in e045600f01dde41e138bc456b9da34bac8f690ce